src/main/scala/org/broadinstitute/hail/driver/AnnotateSamplesExpr.scala

```diff
@@ -28,6 +32,7 @@ object AnnotateSamplesExpr extends Command {
 
   def run(state: State, options: Options): State = {
     val vds = state.vds
+    val branchingFactor = Option(options.branchingFactor).map(x => x: Int)
```

Comment:
do you need to cast here?  Isn't there an implicit conversion that will be invoked below?